src/psyclone/tests/domain/common/algorithm/alg_psyir_test.py

```diff
@@ -661,8 +660,18 @@ def test_aic_lowertolanguagelevel_invoke_symbols_scope():
         Literal("1", INTEGER_TYPE), [invoke.detach()])
     psyir.children[0].children.append(loop)
 
+    assert "invoke" not in invoke.scope.symbol_table._symbols
+    assert "invoke" in loop.scope.symbol_table._symbols
+
     psyir.lower_to_language_level()
 
+    assert len(psyir.walk(AlgorithmInvokeCall)) == 0
```

Comment:
`assert not psyir.walk(xxxx)` again and below.

Reply:
Updated.